docs/users-guide/05-visualizing-results.md

```diff
@@ -140,13 +142,39 @@ If your table is a result that contains one numeric column and two grouping colu
 
 ![Pivot table](images/visualizations/pivot.png)
 
+This auto-pivoting is distinct from the pivot table visualization, which we cover next.
+
+### Pivot tables
+
+Pivot tables allow you swap rows and columns, group data, and include aggregate data in your table. You can group one or more metrics by one or more dimensions. 
```

Comment:
I think we should go with the more expected "subtotals" throughout rather than "aggregate date" even though it's true that there are exceptions like averages where it's not _really_ a total.

```suggestion
Pivot tables allow you swap rows and columns, group data, and include subtotals in your table. You can group one or more metrics by one or more dimensions. 
```